### DISCUSSION TITLE
Just wanted to say thanks for awesome project!

### DISCUSSION THREAD

**Discussion by xseman** [wailsapp/wails#1850] [General]
Hello world!

**Comment by leaanthony** [comment]
Thanks @xseman! It's a huge team effort and a real pleasure to work with. The project got to where it is today through the efforts of many people. We welcome all contributions, no matter how big or small and encourage everyone to contribute. That could be a fix to the documentation, trying to fix a bug, attempting a ticket with the TODO label or simply testing a PR and providing feedback. Let us know what you build 👍
